### PR TITLE
Uncapitalize and disambiguate an advisory (non-normative) media overlays MAY

### PR DESCRIPTION
**Pull request by dlazin**
This isn't giving the RS an option on how to process the content; it's reminding them to watch out for naive assumptions.

### CODE REVIEW

**Review by iherman — APPROVED** [review]

**Comment by iherman** [comment]
~Wait for CR publication, and then the merge of #2268, before merging this!~ #2268 has been merged

**Review by mattgarrish — APPROVED** [review]

**Pull request merged**
